learning_assistant/api.py

```diff
@@ -189,6 +191,24 @@ def get_course_id(course_run_id):
     return course_key
 
 
+def save_chat_message(user_id, chat_role, message):
```

Comment:
This function should also take in a course_id, and use that as a field in the message being saved

Reply:
Nice catch. I wonder why this wasn't detected on the unit tests, maybe there's a constraint missing?

Reply:
Just updated `save_chat_message()` to take the Courserun Key and save it to the model.